apps/schools/domains/common/components/field/index.tsx

```diff
@@ -0,0 +1,33 @@
+import styles from './styles/styles.module.scss'
+import { Col, Row } from 'antd'
+import React from 'react'
+import { FieldProps } from './interface'
+
+export const Field = (props: FieldProps) => {
+    const { fieldName, fieldValue = undefined, defaultValue, type = 'right' } = props
+
+    let lgxs
+
+    if (type === 'left') {
+        lgxs = 14
+    } else if (type === 'center') {
+        lgxs = 12
+    } else {
+        lgxs = 10
+    }
+
+    if (!fieldValue && !defaultValue) {
+        return null
+    }
```

Comment:
make them required in the interface and you won't have to do this code.

Reply:
it is logic for returning nothing if there is no defaultValue and fieldValue
it is useful when you don't want to show fields in which fieldValue's are undefined